example-mojaloop-backend/values.yaml

```diff
@@ -326,48 +197,25 @@ centralledger-obj:
   ## Enable persistence using Persistent Volume Claims
   ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
   ##
-  persistence:
-    ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
-    ##
-    enabled: false
+  # persistence:
+  #   ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
+  #   ##
+  #   enabled: false
```

Comment:
Why is this commented out?

Reply:
Please do.

It doesn't make sense that It wouldn't work when persitence is disabled. The only difference is how the "data" is persisted or not.